roles/bootstrap-os/tasks/bootstrap-ubuntu.yml

```diff
@@ -21,8 +21,11 @@
 
 - name: Bootstrap | Install python 2.x and pip
   raw:
-    apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y {{ubuntu_packages | join(" ")}}
+    (export no_proxy={{proxy_env.no_proxy if (proxy_env is defined and proxy_env.no_proxy is defined) else ""}} \
+    http_proxy={{proxy_env.http_proxy if (proxy_env is defined and proxy_env.http_proxy is defined) else ""}} \
+    https_proxy={{proxy_env.https_proxy if (proxy_env is defined and proxy_env.https_proxy is defined) else ""}} \
```

Comment:
environment should configured in the 'environment' field

Reply:
We can not set env with a "raw" task in ansible.